src/main/scala/org/allenai/pipeline/Pipeline.scala

```diff
@@ -181,6 +215,38 @@ trait ConfiguredPipeline extends Pipeline {
 
     logger.info(s"Summary written to $outputDir")
   }
+}
+
+trait ConfiguredPipeline extends Pipeline {
+  def config: Config
```

Comment:
Not excited about trait args. How about class ConfiguredPipeline(config: Config) instead?